pkg/runtime/runtime.go

```diff
@@ -2154,6 +2169,25 @@ func (a *DaprRuntime) isComponentAuthorized(component componentsV1alpha1.Compone
 	return true
 }
 
+func (a *DaprRuntime) IsComponentLoaded(component componentsV1alpha1.Component) bool {
+	for _, loadedComp := range a.components {
```

Comment:
I guess we need a readlock RLock/RUnlock here, correct?

Reply:
added, thanks!